second-semester/Filesystem/src/main/java/ua/knu/filesystem/FileManager.java

```diff
@@ -188,52 +258,99 @@ public void remove(int filename) {
         }
     }
 
-    public void write(int id, String str) throws FileFullException {
-        Descriptor file = oft.getDescriptorByID(id);
-        OFTEntry entry = ((OpenFileTable) oft).getEntry(id);
+    private void setDirectoryEntryByID(DirectoryEntry entry, int id) throws DirectoryEntryNotFoundException {
```

Comment:
All methods above were just marked as private and moved at the bottom of the class